tyr/servers/mongo/data.py

```diff
@@ -77,18 +77,26 @@ def bake(self):
                 {
                     'user': 'mongod',
                     'group': 'mongod',
-                    'size': 10,
-                    'iops': 0,
+                    'size': self.data_volume_size,
+                    'iops': self.data_volume_iops,
+                    'device': '/dev/xvdf',
+                    'mount': '/volr'
+                },
+                {
+                    'user': 'mongod',
+                    'group': 'mongod',
+                    'size': 25,
+                    'iops': 250,
                     'device': '/dev/xvdg',
                     'mount': '/volr/journal'
                 },
                 {
                     'user': 'mongod',
                     'group': 'mongod',
-                    'size': self.data_volume_size,
-                    'iops': self.data_volume_iops,
-                    'device': '/dev/xvdf',
-                    'mount': '/volr'
+                    'size': 50,
+                    'iops': 250,
```

Comment:
Do we always want IOPS regardless of the environment? Also, would it make sense to have arguments and variables `log_volume_iops` and `log_volume_size`?